modules/ui/field.js

```diff
@@ -34,7 +35,7 @@ export function uiField(context, presetField, entityIDs, options) {
 
     var dispatch = d3_dispatch('change', 'revert');
     var field = Object.assign({}, presetField);   // shallow copy
-    field.domId = utilUniqueDomId('form-field-' + field.safeid);
```

Comment:
Were these methods really functionally equivalent? Weird!

Reply:
oh yeah I renamed:
- `utilSafeClassName` -> `utilSafeString`
- `utilUniqueDomId` -> `utilUniqueString`

They can be used for things other than classnames and domids.